### ISSUE TITLE
Implement views_content_cache for CIS

### ISSUE THREAD

**Issue by mmilutinovic1313** [elmsln/elmsln#1256]
We need to implement `views_content_cache` in CIS to set the caching for these views to be content based and to be set at the highest time increment (6 days). This will be applied to all view types that are visible to users. For example -- Field Collections will not be cached in this way as they will be hit / cached when their content is updated.

**Comment by btopro** [comment]
changed title; this is to fix views caching across all systems

**Comment by mmilutinovic1313** [comment]
Awesome -- After adding the caching, will I be exporting the view like I did the last time we played with caching options?

**Comment by btopro** [comment]
yup, same thing, basically switching them all over to this plugin cause this is soooo much more logical

**Comment by mmilutinovic1313** [comment]
Okay commenting the views from each distro below as a check off list for this task:

**Comment by mmilutinovic1313** [comment]
# MOOC / Course

- [ ] Book Search and Replace (Content)
- [ ] CIS Section list (Content)
- [ ] CIS User list (User)
- [ ] Feeds log (Feeds log)
- [ ] Feeds Node Helper Export (Content)
- [ ] MOOC book roots (Content)
- [ ] OG User groups (OG membership)

**Comment by mmilutinovic1313** [comment]
@btopro: for courses, there is a OG caching option. It has two options of _Current Group_ and _Member's Group_. Would you like me to use either of these to cache the OG views? There aren't too many content-based cacheable views within MOOC/Courses it seems.
